### ISSUE TITLE
Indoor Flowerpots

### ISSUE THREAD

**Issue by jeykloh** [Ottermandias/Accountant#55]
If the table that indoor flowerpots are moved, accountant no longer recognizes any tending for the second or third flowerpots (only tried this in Mediums) to be edited.

Additionally, is there a way to manually edit the timer for the outdoor plots or indoor flowerpots?

**Comment by Ottermandias** [comment]
Yes, the pots are identified by their location. There is no other way to differentiate them that I know of. You can edit the jsons stuff is stored in, of course.
